xilem/src/view/variable_label.rs

```diff
@@ -0,0 +1,176 @@
+// Copyright 2024 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use masonry::{
+    parley::{
+        fontique::Weight,
+        style::{FontFamily, FontStack, GenericFamily},
+    },
+    text::TextBrush,
+    widget, ArcStr,
+};
+use xilem_core::{Mut, ViewMarker};
+
+use crate::{Color, MessageResult, Pod, TextAlignment, View, ViewCtx, ViewId};
+
+pub fn variable_label(label: impl Into<ArcStr>) -> VariableLabel {
```

Comment:
I wonder (not necessarily relevant for this PR), if this should be a "transformer" modifier on the `Label` (i.e. moves all relevant values of `label()` to this type). (And not a separate view constructor). 

so that we could do something like this instead:

```rust
label("label").target_weight(..)
```

(Not sure if more complex typing to achieve this with traits etc. makes sense here)

Reply:
Oh I might have missed that, nice.